docs/source/getting_started/cpu-installation.rst

```diff
@@ -56,7 +56,7 @@ Build from source
 .. code-block:: console
 
     $ pip install --upgrade pip
-    $ pip install wheel packaging ninja "setuptools>=49.4.0" numpy
+    $ pip install cmake>=3.6 wheel packaging ninja "setuptools-scm>=8" numpy
```

Comment:
`3.6` or `3.26`?

Reply:
Good catch, fixed in the last force push